.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "bracketSameLine": false
```

Comment:
```diff
{
-   "bracketSameLine": false
+   "bracketSameLine": true
}
```

The `"bracketSameLine": false` is the default behaviour, by having the ending bracket on the same line makes the code look clean.

Reply:
updating